### PR TITLE
Added Uses to Cactus/Sugar Charcoal/Coke

### PR DESCRIPTION
**Pull request by Steelux8**
- Changed the existing Industrial Coke Oven recipes to make them a lot faster than the Charcoal recipes, since they only make 1 at a time compared to the Charcoal recipes' 5;
- Added uses to these items, such as Charcoal Byproducts output, Wood Tar extraction, Coal Tar and Gas, and also CO and CO2.

As per https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10233, Charcoal and Coal Coke can be made from Cactus and Sugar Cane, although with less fuel value than the original versions. However, these don't have most of the uses that those do, being mostly used as furnace fuel and little else. With this PR, I changed and created recipes so that these can be used similarly to Wood and Charcoal, although with less available uses and automations, which leave Wood and Charcoal as the best options after the first couple of tiers.

### CODE REVIEW

**Comment by Steelux8** [comment]
Still waiting for approval on this to close the issue mentioned above.

File: src/main/java/gtPlusPlus/xmod/railcraft/HANDLER_Railcraft.java
Comment (GlodBlock) [inline src/main/java/gtPlusPlus/xmod/railcraft/HANDLER_Railcraft.java]: remove coal tar/gas recipe or change them into wood tar/gas.

**Review by GlodBlock — CHANGES_REQUESTED** [review]

File: src/main/java/gtPlusPlus/xmod/railcraft/HANDLER_Railcraft.java
Comment (Steelux8) [reply src/main/java/gtPlusPlus/xmod/railcraft/HANDLER_Railcraft.java]: It's done. I liked the renewable Coal Tar and Gas, but Coal itself is already renewable, and making the Wood versions from these makes more sense.

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**
